tf_files/aws/commons/variables.tf

```diff
@@ -172,12 +188,76 @@ variable "mailgun_smtp_host" {
 variable "mailgun_api_url" {
   default = "https://api.mailgun.net/v3/"
 }
+<<<<<<< HEAD
```

Comment:
I think there is some sort of conflict with your branch, this looks like you merged two branches and it generated some git merging garbage.